src/os/bluestore/BlueFS.cc

```diff
@@ -3215,8 +3220,7 @@ int BlueFS::_flush_and_sync_log_LD(uint64_t want_seq)
 }
 
 // Flushes log and immediately adjusts log_writer pos.
-int BlueFS::_flush_and_sync_log_jump_D(uint64_t jump_to,
-				     int64_t available_runway)
+int BlueFS::_flush_and_sync_log_jump_D(uint64_t jump_to, bool ignore_compaction)
```

Comment:
Not exactly. `log_forbidden_to_expand` is what says whether a compaction is running and `ignore_compaction` is a way of telling ignore it because we are the ones running compaction. Anyways, I'll change it so that we don't need it.

Reply:
Now, I guess after some changed, `ignore_compaction` is unused at all.